indicluster/app.py

```diff
@@ -182,26 +192,52 @@ def post(self):
                 'error': "rss feed missing standard field 'title' and/or 'link'"
             })
 
-        text = POOL.map(_read_text_from_url, links)
+        text = map(_read_text_from_url, links)
         objs = zip(links, titles, text)
         # obj[2] --> text
         objs = filter(lambda obj: obj[2].strip() != "", objs)
         text = [item[2] for item in objs] # no longer contains empty strings
-        indico = indicoio.analyze_text(
-            text,
-            apis=['text_features', 'people', 'places', 'organizations']
-        )
-        keywords = indicoio.keywords(text, version=2, top_n=3)
+        if not text:
+            print "URL", url
+            print "LINKS", links
+            return self.write(json.dumps({}))
+
+        text_features = []
```

Comment:
I would make these 7 lists a dictionary of api_name to list. It's really clunky having 7 lists here.